### ISSUE TITLE
Error occurred when executing SEGSDetailerForAnimateDiff

### ISSUE THREAD

**Issue by Zaithe** [ltdrdata/ComfyUI-Impact-Pack#352]
Hello, when i try to use the detailer for AnimatedDiff i get the below error:
```
Error occurred when executing SEGSDetailerForAnimateDiff:

The expanded size of the tensor (72) must match the existing size (73) at non-singleton dimension 3. Target sizes: [1, 4, 96, 72]. Tensor sizes: [4, 97, 73]

File "F:\ComfyUI_windows_portable\ComfyUI\execution.py", line 153, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "F:\ComfyUI_windows_portable\ComfyUI\execution.py", line 83, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "F:\ComfyUI_windows_portable\ComfyUI\execution.py", line 76, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "F:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\modules\impact\segs_nodes.py", line 197, in doit
segs = SEGSDetailerForAnimateDiff.do_detail(image_frames, segs, guide_size, guide_size_for, max_size, seed, steps, cfg, sampler_name,
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "F:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\modules\impact\segs_nodes.py", line 178, in do_detail
enhanced_image_tensor = core.enhance_detail_for_animatediff(cropped_image_frames, model, clip, vae, guide_size, guide_size_for, max_size,
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "F:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\modules\impact\core.py", line 359, in enhance_detail_for_animatediff
samples = to_latent_image(upscaled_image, vae)['samples']
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "F:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\modules\impact\utils.py", line 262, in to_latent_image
t = vae.encode(pixels[:, :, :, :3])
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "F:\ComfyUI_windows_portable\ComfyUI\comfy\sd.py", line 259, in encode
samples[x:x+batch_number] = self.first_stage_model.encode(pixels_in).to(self.output_device).float()
~~~~~~~^^^^^^^^^^^^^^^^^^
```
I looked at previous posts to try to troubleshoot and did what they recommended by setting sam to load with CPU and reverting pillow to 9.5 but those didn't work. I can send the workflow to a person willing to help me troubleshoot. I've been trying to resolve it for two nights now and am running out of ideas.

**Comment by ltdrdata** [comment]
What is your workflow?

**Comment by Zaithe** [comment]
> What is your workflow?

Attached: 
[DetailerAnimatedDiff.json](https://github.com/ltdrdata/ComfyUI-Impact-Pack/files/13623261/DetailerAnimatedDiff.json)

**Comment by ltdrdata** [comment]
This is fixed on V4.41.1.

"taesd/taesdxl" requires a more strict image size, and there was an error because it did not meet the specified requirements.
